Artur Zakharian/Task4.3.py

```diff
@@ -0,0 +1,9 @@
+friends = ["John", "Marta", "James"]
+enemies = ["John", "Jonathan", "Artur"]
+for i in friends:
+    if i == "James":
+        print(i, "we best friends")
```

Comment:
please read the task carefully. we should not check James

Reply:
fixed